webviz_config/webviz_deprecated.py

```diff
@@ -0,0 +1,91 @@
+from typing import Dict, Union, Callable, Tuple, cast, Optional, List
+import inspect
+import warnings
+
+from . import _plugin_abc
```

Comment:
```suggestion
from ._plugin_abc import WebvizPluginABC
```
And then `_plugin_abc.WebvizPluginABC` can be replaced with `WebvizPluginABC` below?